src/app/components/modules/simplified-lunch/simplified-lunch.component.ts

```diff
@@ -0,0 +1,118 @@
+import { Component, Input, OnInit, OnDestroy, ElementRef, ViewChild } from '@angular/core';
```

Comment:
You've got some unused imports--specifically `Input`, `ElementRef` and `ViewChild`